src/main/resources/assets/computercraft/lua/rom/programs/copy.lua

```diff
@@ -10,7 +10,9 @@ local sDest = shell.resolve( tArgs[2] )
 local tFiles = fs.find( sSource )
 if #tFiles > 0 then
     for n,sFile in ipairs( tFiles ) do
-        if fs.isDir( sDest ) then
+        if fs.exists( sDest ) == true then
```

Comment:
This check is incorrect. "copy existingfile existingdir" is perfectly valid, hence the isDir() check on the very next line, please move it inside the "#tFiles == 1" branch and remove the "== true"